tests/unit/test__helpers.py

```diff
@@ -806,6 +806,35 @@ def test_w_known_field_type(self):
         self.assertEqual(converted, str(original))
 
 
+class Test_single_field_to_json(unittest.TestCase):
+    def _call_fut(self, field, value):
+        from google.cloud.bigquery._helpers import _single_field_to_json
+
+        return _single_field_to_json(field, value)
+
+    def test_w_none(self):
+        field = _make_field("INT64")
+        original = None
+        converted = self._call_fut(field, original)
+        self.assertIsNone(converted)
+
+    def test_w_record(self):
+        subfields = [
+            _make_field("INT64", name="one"),
```

Comment:
The main point of the `_single_field_to_json` function is that it ignores the `REPEATED` property, right? We should have tests that validate this behavior (that is: set mode = REPEATED in some fields)